Sources/DittoAllToolsMenu/Views/Identity Configuration View/IdentityFormData.swift

```diff
@@ -0,0 +1,131 @@
+// 
+//  IdentityFormData.swift
+//
+//  This file defines a data model used to handle the form data in the identity configuration process.
+//  It provides methods to initialize form data from existing identity configurations and to convert form data into a Ditto `IdentityConfiguration`.
+//
+//  Copyright © 2024 DittoLive Incorporated. All rights reserved.
+//
+
+import SwiftUI
+import DittoSwift
+
+/// A data model that stores the identity configuration data entered in the form.
+///
+/// `IdentityFormData` contains all fields necessary for configuring different types of identities used in Ditto.
+/// It supports various identity types, such as `onlinePlayground`, `offlinePlayground`, and `manual`. This struct
+/// also provides functionality to convert between form data and `IdentityConfiguration` instances.
+struct IdentityFormData {
+    var identityType: DittoIdentity.IdentityType = .onlinePlayground
+    var appID: String = ""
+    var offlineLicenseToken: String = ""
+    var playgroundToken: String = ""
+    var enableDittoCloudSync: Bool = true
+    var authProvider: String = ""
+    var authToken: String = ""
+    var customAuthURL: String = ""
+    var siteID: UInt64 = 0
```

Comment:
If possible, I'd like to intentionally leave siteID out of all our demo apps & tools, as we would rarely if ever recommend customer set it themselves, and it has been mostly replaced by `peerKey`